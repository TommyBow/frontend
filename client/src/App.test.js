import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import TaskForm from './TaskForm';

test('submit button works', () => {
  const { getByText } = render(<TaskForm />);
  const submitButton = getByText('Submit');
  fireEvent.click(submitButton);
});
