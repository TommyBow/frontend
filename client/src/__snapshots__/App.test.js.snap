// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the TaskForm component 1`] = `
<DocumentFragment>
  <form
    class="task-form"
  >
    <input
      placeholder="Note Title"
      required=""
      type="text"
      value=""
    />
    <textarea
      placeholder="Add description"
      required=""
    />
    <label
      for="task-date"
    >
      Due Date:
    </label>
    <input
      id="task-date"
      required=""
      type="date"
      value=""
    />
    <button
      type="submit"
    >
      Add Note
    </button>
  </form>
</DocumentFragment>
`;
